tensorboard/webapp/core/store/core_types.ts

```diff
@@ -29,9 +29,22 @@ export interface CoreState {
   pluginsListLoaded: LoadState;
   reloadPeriodInMs: number;
   reloadEnabled: boolean;
+  // Size of a page in a paginated view.
```

Comment:
nit: A paginated view of what?

Reply:
It is not defined concretely on purpose. It can be used for many UI elements.

Reply:
Understood. Just state the fact that it's meant for general UI element use in the comment explicitly. :)